src/components/App/index.tsx

```diff
@@ -1,20 +1,26 @@
-import React, { useState } from 'react';
+import React, { useEffect } from 'react';
 import i18next from 'i18next';
 
+import { Context, reducer, INITIAL_STATE, useSelector } from 'contexts';
```

Comment:
The context must not be 'index.ts' you should call it UserContext.ts or something similar